<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlBlock</class>
 <widget class="QFrame" name="ControlBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pb_start">
     <property name="toolTip">
      <string>Start</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_pause">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Pause</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-playback-pause"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Stop</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="media-playback-stop"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_mode"/>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
